book/website/ethical-research/data-feminism/examining-power.md

```diff
@@ -0,0 +1,74 @@
+(er-data-feminism-c1)=
+
+# Chapter 1: Examining Power
+
+Here are some main ideas from this chapter on examining power. 
+
+(er-data-feminism-c1-keypoints)=
+## Key Points
+- **Understanding Influence**: It is essential to recognize how racism, sexism, and privilege shape oppression.
+- **Naming Oppression**: Examining power involves identifying and explaining the oppressive forces ingrained in our daily lives.
+
+(er-data-feminism-c1-powerdomains)=
+## Domains of Power
+Power systems can be configured and experienced across four domains:
+
+![image](https://hackmd.io/_uploads/BJeHBss40.png)
+*Source: https://data-feminism.mitpress.mit.edu/pub/vi8obxh7#nkiyehtj87r*
```

Comment:
Could you follow the [style guide for figures](https://book.the-turing-way.org/community-handbook/style/style-figures#using-figures-in-the-turing-way) in the Community Handbook?

As it is a table, but might be better to reproduce it in the book [as a table](https://jupyterbook.org/en/stable/reference/cheatsheet.html#tables) and include a reference.

Reply:
Hi! The matrix wasn't particularly necessary as everything was listed out anyway, so I took it out.